packages/react-native/project.json

```diff
@@ -31,6 +55,20 @@
           "cp packages/react-native/src/NativeBdReactNative.ts dist/react-native/src"
         ],
         "parallel": false
+      },
+      "outputs": ["{workspaceRoot}/dist"]
+    },
+    "TODOrelease": {
```

Comment:
Do we still want to call this TODOrelease?

Would also be nice to have this hooked up with the release github action that we use for electron if its not too much work

Reply:
I was leaving it as TODO until I actually had time to look at hooking into the electron release, we'll probably need to add a bunch of secrets to get this usable in CI.